clients/fides-js/tsconfig.json

```diff
@@ -13,10 +15,11 @@
     "skipLibCheck": true,
     "strict": true,
     "target": "es2015",
-
+    "plugins": [{ "name": "typescript-plugin-css-modules" }],
     "baseUrl": "./",
     "paths": {
-      "~/*": ["src/*"]
+      "react": ["./node_modules/preact/compat/"],
+      "react-dom": ["./node_modules/preact/compat/"]
```

Comment:
Curious, what are these paths used for?

Reply:
After all I think the `preact` module would be up in the `clients/` folder, so `../node_modules/...`?

Reply:
this was probably leftover from testing, and can be removed.